Source/CurtisInstruments.CaliburnMicroDevExpress.Extensions/CurtisInstruments.CaliburnMicroDevExpress.Extensions.csproj

```diff
@@ -3,10 +3,10 @@
 
   <PropertyGroup>
     <AssemblyName>CurtisInstruments.CaliburnMicroDevExpress.Extensions</AssemblyName>
-    <TargetFrameworks>net452;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks>net472;net5.0-windows</TargetFrameworks>
```

Comment:
Unless there's a requirement to the contrary, it's probably better to leave this at `net452` in case anyone on an older framework version wants to use it.

Reply:
No reason. DX still supports 4.5.2 I'll revert it.